CUSTOM_BUNDLE.md

```diff
@@ -1,13 +1,29 @@
 # Custom bundle
 You can simply make custom bundles yourself, if none of the [distributed packages](https://github.com/plotly/plotly.js/blob/master/dist/README.md) meet your needs, or you want to make a more optimized bundle file with/without specific traces and transforms.
 
-Install plotly.js, move to plotly.js folder then install plotly.js dependencies:
+Clone plotly.js, where the <version> is one of [these](https://github.com/plotly/plotly.js/tags):
+```sh
+git clone --depth 1 --branch <version> https://github.com/plotly/plotly.js.git
+```
+
+Move to plotly.js folder then install plotly.js dependencies:
+```sh
+cd plotly.js
+```
+
+Make sure you have the versions of node/npm that's recommended in the [./CONTRIBUTING.md](./CONTRIBUTING.md#prerequisites) list of Prerequisites
```

Comment:
What should they do if they don't have the right versions?
If they installed node via `nvm`, that's quite easy to fix.
If not it would be difficult. No?

Reply:
If they use fnm (linux/win) or nvm (mac) it's easy enough, and [the recommendation](https://nodejs.org/en/download/package-manager/current), and if they use a prebuilt binary, they they download another prebuilt binary.

Reply:
OK. Let's provide this information there.